crates/sdk/src/rpc.rs

```diff
@@ -1167,25 +1166,9 @@ pub async fn validate_amount<N: Namada>(
         InputAmount::Unvalidated(amt) => amt.canonical(),
         InputAmount::Validated(amt) => return Ok(amt),
     };
-    let base_token =
-        if let Address::Internal(InternalAddress::IbcToken(ibc_token_hash)) =
-            token
-        {
-            extract_base_token(context, ibc_token_hash.clone(), None).await
```

Comment:
There is no guarantee that the base token of the source chain is the same as that of the destination(this) chain.
We should use zero denomination for all `IbcToken`s.